### PR TITLE
[CoastalArea] Fix broken link to Algarve bathymetry

### PR DESCRIPTION
**Pull request by david451lima**
Fix broken link
Fixed a type

### CODE REVIEW

File: inductiva/coastal/README.md [hunk 1]
```diff
@@ -79,13 +79,13 @@ Now let's perform our simulations with real data!
 Start by loading a bathymetry. This can be done e.g. by reading an
 [ASCII XYZ file](https://emodnet.ec.europa.eu/sites/emodnet.ec.europa.eu/files/public/20171127_DTM_exchange_format_specification_v1.6.pdf),
 a standard format for bathymetry data (for examples, see e.g. the [European Marine Observation and Data Network](https://emodnet.ec.europa.eu/geoviewer/#!/)).
-Here we shall use the bathymetry data from Algarve that you can download [here](/resources/bathymetry/590_HR_Lidar_Algarve.emo)
+Here we shall use the bathymetry data from Algarve that you can download [here](https://downloads.emodnet-bathymetry.eu/high_resolution/590_HR_Lidar_Algarve.emo.zip)
```
Comment (hpenedones) [inline inductiva/coastal/README.md:82]: you're downloading a zip file?

Comment (IvanPombo) [reply inductiva/coastal/README.md:82]: Try to follow Hugo tip and use the download from url - see #606 - to obtain the file in the code.

Comment (hpenedones) [reply inductiva/coastal/README.md:82]: Why, and why? (For those two statements). Can you explain further?

Comment (david451lima) [reply inductiva/coastal/README.md:82]: Luís is helping me with a download_url function that can get and unzip files! That will solve this issue

File: inductiva/coastal/README.md [hunk 2]
```diff
@@ -79,14 +79,18 @@ Now let's perform our simulations with real data!
...
-as an example.
+Here we shall use the bathymetry data from Algarve.
 
 ```python
 import inductiva
 
+bathymetry_url = "https://downloads.emodnet-bathymetry.eu/high_resolution/590_HR_Lidar_Algarve.emo.zip"
+
+bathymetry_path = inductiva.utils.files.download_from_url(bathymetry_url,
+local_file_path = "590_HR_Lidar_Algarve.emo")
```
Comment (IvanPombo) [inline inductiva/coastal/README.md:90]: I think you can drop this to simplify!

**Review by IvanPombo — APPROVED** [review]
LGTM! See my only comment to simplify.

File: inductiva/coastal/README.md (hunk 2 above)
Comment (david451lima) [reply inductiva/coastal/README.md:90]: Sweet!

**Pull request merged**